src/Components/DraggableResizer/DraggableResizer.tsx

```diff
@@ -131,7 +133,9 @@ export const DraggableResizerRoot: FunctionComponent<DraggableResizerProps> = ({
 
   const calculateCollapsePosition = (direction: number, dragIndex: number) => {
     const totalDrag = handlePositions.length - 1
-    const directionIndex = direction === 0 ? dragIndex : dragIndex + 2
+    const directionOffset = PANEL_COUNT % 2 == 0 ? 2 : 3
```

Comment:
Just wondering, what does `2` and `3` mean here?

Reply:
It's an offset on top of the drag index.
So the index is 0 - X depending on the number of handles. We add this offset to make it 0 - X + 2 if its even or 0 -X + 3 if its odd. 

This summed value is then multiplied against 0.25 (which is each individual breakpoint for the panels). And that's what pushes it closer to the 0 or 1 value that is needed to collapse it.

This component is extremely complicated hah